crypto/rsa/rsa_gen.c

```diff
@@ -425,12 +425,13 @@ static int rsa_keygen(OSSL_LIB_CTX *libctx, RSA *rsa, int bits, int primes,
                       BIGNUM *e_value, BN_GENCB *cb, int pairwise_test)
 {
     int ok = 0;
+    int ebits = e_value != NULL ? BN_num_bits(e_value) : 17;
 
     /*
      * Only multi-prime keys or insecure keys with a small key length will use
```

Comment:
maybe also update the comment?
_...insecure keys with a small key length **or a public exponent <= 2^16** will use ..._